src/components/PlaygroundPage/PlaygroundPage.js

```diff
@@ -169,8 +169,8 @@ const PlaygroundPage = () => (
       </CircularButton>
       <Card />
       <Field placeholder="eg... http:zombo.com" />
-      <Field icon={DashboardIcon} placeholder="eg... http:zombo.com" />
-      <Field icon={DashboardIcon} text="EST" placeholder="Time" />
+      <Field leftIcon={DashboardIcon} placeholder="eg... http:zombo.com" />
```

Comment:
```suggestion
      <Field leftIcon={DashboardIcon} placeholder="eg... http://zombo.com" />
```

Although we should probably replace instances of zombo.com with example.com or something like that.

Reply:
Yeah, also it is just the `playground` page used only for examples.